lib/stackConfiguration/src/main/resources/stack-configuration.spb.xml

```diff
@@ -15,53 +15,35 @@
 	factory-method="singleton" scope="singleton" />
 	
 	<bean id="stackConfiguration.stackAndStackInstancePrefix" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[LOCATIONABLE_CONVERT_TYPE]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[APPEND_ROW_SET_TO_TABLE]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[UPLOAD_CSV_TO_TABLE]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[UPLOAD_CSV_TO_TABLE_PREVIEW]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[DOWNLOAD_CSV_FROM_TABLE]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[QUERY]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[QUERY_NEXT_PAGE]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[OBJECT]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[PRINCIPAL_PREFIX]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[S3_FILE_COPY]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[BULK_FILE_DOWNLOAD]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[BROADCAST]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[PROJECT_STATS_V2]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[TABLE_UPDATE_TRANSACTION]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[THREAD_STATS]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[MIGRATION]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[DOI]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
-	<bean name="stackConfiguration.asyncQueueName[ADD_FILES_TO_DOWNLOAD_LIST]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />
+	<bean name="stackConfiguration.queueName[LOCATIONABLE_CONVERT_TYPE]" class="org.springframework.beans.factory.config.PropertyPathFactoryBean" />

```

Comment:
investigate and try to remove usage of these beans. workers should know the exact queue they need and directly call stackconfiguration